mobile-widgets/qml/DiveList.qml

```diff
@@ -421,6 +421,16 @@ Kirigami.ScrollablePage {
 							forceActiveFocus()
 						}
 					}
+					onFocusChanged: {
```

Comment:
This block gives another issue, the keyboard is not hidden when pressing to hide the filter view.
Removing the onFocusChanged block solves this issue for me.

Reply:
That's wild since I actually ask to hide it RIGHT THERE.
But without this block I could easily reproduce a situation where I had no keyboard and tapping on the input field didn't get me a keyboard, either. So I'm not sure what to do here... maybe don't try to explicitly hide it?

Reply:
Yeah, that's what bugs me the most. How can we write an app that works for all (or at least most) of our users???
Not sure what to do here - is there another way to ensure that the keyboard is shown when the TextField is entered?

Reply:
Activating the textfield by tapping on it will always bring up the keyboard in my tests. And the forceActiveFocus fixes the issue with displaying the keyboard when opening the filter view.